pincer/objects/guild/guild.py

```diff
@@ -601,18 +602,14 @@ async def create_role(
         """
         ...
 
-    async def create_role(
-        self,
-        reason: Optional[str] = None,
-        **kwargs
-    ) -> Role:
+    async def create_role(self, reason: Optional[str] = None, **kwargs) -> Role:
         return Role.from_dict(
             construct_client_dict(
                 self._client,
                 await self._http.post(
                     f"guilds/{self.id}/roles",
                     data=kwargs,
-                    headers=remove_none({"X-Audit-Log-Reason": reason})
+                    headers=remove_none({"X-Audit-Log-Reason": reason}),
```

Comment:
no comma, I dont think we should listen to black for this one rule

Reply:
@Sigmanificient If so, find a way to remove trailing commas from black. We cannot always have to remove the commas whenever we format. It would be tedious especially as the project grows.